<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderWindow</class>
 <widget class="QWidget" name="OrderWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>721</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>721</width>
     <height>671</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Куда нужно привезти?</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="DepartureComboBox"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Груз</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Суммарный вес груза</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Заказчик</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="ArrivalComboBox"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Откуда нужно отвезти?</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QComboBox" name="CargoTypeComboBox"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="CustomerEdit"/>
    </item>
    <item row="5" column="1">
     <widget class="QComboBox" name="CargoComboBox"/>
    </item>
    <item row="0" column="1">
     <widget class="QDateEdit" name="OrderDateEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Тип груза</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QSpinBox" name="WeightSpinBox">
      <property name="maximum">
       <number>100000000</number>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Дата заказа</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="2">
     <widget class="QPushButton" name="OrderButton">
      <property name="text">
       <string>Оформить заказ</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>OrderDateEdit</tabstop>
  <tabstop>CustomerEdit</tabstop>
  <tabstop>DepartureComboBox</tabstop>
  <tabstop>ArrivalComboBox</tabstop>
  <tabstop>CargoTypeComboBox</tabstop>
  <tabstop>CargoComboBox</tabstop>
  <tabstop>WeightSpinBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
